src/models/parsimonious.py

```diff
@@ -36,7 +36,7 @@ def predict(self) -> Tuple[Dict[str, Dict[str, np.ndarray]], Dict[str, np.ndarra
                     print('Target variable not in prediction data!')
                     raise e
 
-                preds_dict[key] = val.x[:, -1, [target_idx]]
+                preds_dict[key] = val.x.historical[:, -1, [target_idx]]
```

Comment:
so this model ONLY has `historical` information right?

Reply:
right (since its predictions are the VHI for the previous month).